### ISSUE TITLE
How to disable classification branch 

### ISSUE THREAD

**Issue by undayo** [matterport/Mask_RCNN#1953]
Need hlep the project I'm working on, I need masks and bounding boxes for my objects. Is it possible to simplify the architecture by removing the part of the classification?

Thank you!

**Comment by Chakib-Belafdil** [comment]
What I am understand from your question is to know how to use the Mask_RCNN for segmentation only.

To do so you only need to set the 'NUM_CLASSES' attribute to 1 (background) + 1 (object you want to detect). There is no such thing as "removing the classification part".

Hope I helped

**Comment by undayo** [comment]
@Chakib-Belafdil If i understand you well, it is  impossible to disable the classifier branch?

**Comment by Chakib-Belafdil** [comment]
No, what I meant is for your need you only have to set NUM_CLASSES = 2 and the network will output only one class ie. your object you want to segment.

You understand now that a classifier that outputs only one class becomes a segmentation algorithm.
